src/domain/common/errors/domain-custom-exception.ts

```diff
@@ -0,0 +1,14 @@
+import { HttpException } from '@nestjs/common';
+import { DomainErrorCode } from './domain-error-code';
+import { getErrorMessage } from 'src/support/error-message.util';
+
+export class DomainCustomExceptionn extends HttpException {
```

Comment:
```suggestion
export class DomainCustomException extends HttpException {
```
@smileboy0014  오타 있습니다 경태님!